app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,342 @@
+            app:layout_constraintStart_toEndOf="@id/guideline_start" />
+
+        <ImageView
+            android:id="@+id/review_star2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star_active"
+            app:layout_constraintTop_toTopOf="@+id/review_star1"
+            app:layout_constraintBottom_toBottomOf="@id/review_star1"
+            app:layout_constraintEnd_toStartOf="@+id/review_star3"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/review_star1" />
+
+        <ImageView
+            android:id="@+id/review_star3"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star_active"
+            app:layout_constraintTop_toTopOf="@+id/review_star1"
+            app:layout_constraintBottom_toBottomOf="@id/review_star1"
+            app:layout_constraintEnd_toStartOf="@+id/review_star4"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/review_star2" />
+
+        <ImageView
+            android:id="@+id/review_star4"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star_active"
+            app:layout_constraintTop_toTopOf="@+id/review_star1"
+            app:layout_constraintBottom_toBottomOf="@id/review_star1"
+            app:layout_constraintEnd_toStartOf="@+id/review_star5"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/review_star3" />
+
+        <ImageView
+            android:id="@+id/review_star5"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="3dp"
+            android:src="@drawable/ic_star_active"
+            app:layout_constraintTop_toTopOf="@+id/review_star1"
+            app:layout_constraintBottom_toBottomOf="@id/review_star1"
+            app:layout_constraintEnd_toStartOf="@+id/number_of_reviews"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintStart_toEndOf="@+id/review_star4"
+            app:tint="@color/inactive_star_color" />
+
+        <TextView
+            android:id="@+id/number_of_reviews"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginStart="8dp"
+            android:text="@string/_125_reviews"
+            android:textAllCaps="true"
+            android:textColor="@color/third_main_color"
+            app:layout_constraintTop_toTopOf="@+id/review_star1"
```

Comment:
maybe vice a versa place review_star1 depending on number_of_reviews by top and bottom?